pages/common/jrnl.md

```diff
@@ -0,0 +1,23 @@
+# jrnl
+
+> A simple journal application for your command line.
+
+- Insert a new entry with your editor:
+
+`jrnl`
+
+- Quickly insert a new entry:
+
+`jrnl {{timestamp}}: {{title}}. {{content}}`
+
+- View the last ten entries:
+
+`jrnl -n {{10}}`
+
+- View everything that happened from the start of last year to the start of last march:
+
+`jrnl -from {{"last year"}} -until {{march}}`
+
+- Edit all entries tagged with @texas and @history:
+
+`jrnl {{@texas}} -and {{@history}} --edit`
```

Comment:
I guess the tags are "texas", "history" and not "@texas", "@history". In that case, please move the @ out of the token. And also, we can remove the @ from the descriptions.

Reply:
Yes, you're right but `-and` accepts other filters and not only tags. I guess to remove the @ from the description but i left @ inside the token

Reply:
Cool, let's do that.